src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs

```diff
@@ -82,46 +79,36 @@ public static void KeepAliveInterval_Roundtrips()
             AssertExtensions.Throws<ArgumentOutOfRangeException>("value", () => cws.Options.KeepAliveInterval = TimeSpan.MinValue);
         }
 
-        [OuterLoop] // TODO: Issue #11345
-        [ActiveIssue(5120, TargetFrameworkMonikers.Netcoreapp)]
-        [ConditionalFact(nameof(WebSocketsSupported), nameof(CanTestClientCertificates), nameof(ClientCertificatesSupported))]
+        [OuterLoop("Connects to remote service")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Lacks RemoteCertificateValidationCallback to enable loopback testing")]
```

Comment:
Weren't we already testing this against NETFX?  Now, by adding this SkipOnTargetFramework, we are no longer testing this against NETFX.

Reply:
Not really.  It's currently skipped for me locally:
```
  call xunit.console.exe System.Net.WebSockets.Client.Tests.dll  -noshadow -xml testResults.xml -notrait category=nonnetfxtests -notrait category=nonwindowstests  -notrait category=failin
  g
  popd
  ===========================================================================================================
  xUnit.net Console Runner (64-bit Desktop .NET 4.0.30319.42000)
    Discovering: System.Net.WebSockets.Client.Tests
    Discovered:  System.Net.WebSockets.Client.Tests
    Starting:    System.Net.WebSockets.Client.Tests
      System.Net.WebSockets.Client.Tests.ClientWebSocketOptionsTests.ClientCertificates_ValidCertificate_ServerReceivesCertificateAndConnectAsyncSucceeds [SKIP]
        Condition(s) not met: "CanTestClientCertificates"
    Finished:    System.Net.WebSockets.Client.Tests
```
and in CI, and when I force the test to run by removing the "Conditional", on netfx I get:
```   System.Net.WebSockets.Client.Tests.ClientWebSocketOptionsTests.ClientCertificates_ValidCertificate_ServerReceivesCertificateAndConnectAsyncSucceeds [FAIL]
        System.Net.WebSockets.WebSocketException : Unable to connect to the remote server
        ---- System.Net.WebException : The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel.
        -------- System.Security.Authentication.AuthenticationException : The remote certificate is invalid according to the validation procedure.
```
even after installing the cert mentioned in the tests.